app/lib/katello/resources/candlepin.rb

```diff
@@ -127,7 +123,7 @@ def create(env_id, _key, name, type, facts, installed_products = [], autoheal =
                      :capabilities => capabilities,
                      :guestIds => guest_ids,
                      :lastCheckin => last_checkin}
-            url += "?activation_keys=" + activation_key_ids.join(",") if activation_key_ids.length > 0
+            url += "?activation_keys=" + activation_keys.join(",") if activation_keys.length > 0
```

Comment:
I would maybe rename the argument to activation_key_ids  or activation_key_cp_ids if you're changing what is being passed in.